centos.sh

```diff
@@ -102,49 +104,49 @@ generate_new_ramdisk() {
     # pick the latest kernel
     VERSION="$(ls -r -1 $FOLD/hdd/boot/vmlinuz-* | head -n1 |cut -d '-' -f 2-)"
 
-    if [ $IMG_VERSION -lt 60 ] ; then
+    if [ "$IMG_VERSION" -lt 60 ] ; then
       MODULESFILE="$FOLD/hdd/etc/modprobe.conf"
```

Comment:
Oh you were talking about MODULESFILE, I meant IMG_VERSION. I will implement the declare -r